lib/commands/app/create.ts

```diff
@@ -33,20 +34,23 @@ interface ArgsDef {
 
 export default class AppCreateCmd extends Command {
 	public static description = stripIndent`
-		Create an application.
+		You can specify the organization the application should belong to using
```

Comment:
First line got accidentally deleted I think... (check how it looks in `cli.markdown`)